bin/http-server

```diff
@@ -38,6 +39,7 @@ if (argv.h || argv.help) {
     '  -K --key     Path to ssl key file (default: key.pem).',
     '',
     '  -r --robots  Respond to /robots.txt [User-agent: *\\nDisallow: /]',
+    '  -v --verbose Log out the full headers of each request along with the typical content',
```

Comment:
We now have `-v` implemented as the short option for `--version`, which I think is fine. This option could be just `--verbose` with no short version.

On the other hand, since http-server is pretty verbose by default, I wonder if this might be better called `--very-verbose`?